pkgs/development/tools/rust/bindgen/hook.sh

```diff
@@ -0,0 +1,9 @@
+# populates LIBCLANG_PATH and BINDGEN_EXTRA_CLANG_ARGS for rust projects that depend on the bindgen crate
```

Comment:
As this hook is for the `bindgen` crate and not the `bindgen` package it may make for sense for it to live with the other rust hooks in `pkgs/build-support/rust/hooks`?

Reply:
the logic shares a lot with the sibling wrapper.sh, so it makes sense to me to leave them together. But if you prefer it this way, I can change.

Reply:
I'd rather it live in `pkgs/build-support/rust` as it is documented in the rust packaging docs.

Could as a comment to each that they share logic?

Reply:
done